SU-SWS/stanford_profile
# D8CORE-3476 Create a new view display mode specific for viewfields

# READY FOR REVIEW

# Summary
- use the custom `viewfield_block` so that we can have different blocks allowed in the viewfield fields.
- It doesn't change any underlying functionality, it just allows for 2 identical block categories.

# Need Review By (Date)
- 3/3

# Urgency
- medium

# Steps to Test
1. checkout branches
   - this branch
   - branch in https://github.com/SU-SWS/stanford_fields/pull/14
   - branch in https://github.com/SU-SWS/stanford_profile_helper/pull/77
2. drush cr; drush cim -y
3. create a basic page and ensure the list paragraph options are still the same as they were before.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
